### PR TITLE
Add Dynamic-Dark-Mode v1.0.6, a dark mode modulator for mac

### LINKED ISSUE #32: Homebrew Cask Support
I just realized that we only need 50 stars to be included. I’m thinking about using `dynamic-dark-mode` as the cask name.

### PR DESCRIPTION
**Pull request by dgisser**
closes ApolloZhu/Dynamic-Dark-Mode#32

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/dynamic-dark-mode.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'dynamic-dark-mode' do
+  version '1.0.6'
+  sha256 '2e82507ad65cf4eda208a8f41b4fe54944d0c04d15d1a49d1bcf546c65b7454d'
+
+  url "https://github.com/ApolloZhu/Dynamic-Dark-Mode/releases/download/#{version}/Dynamic-#{version}.zip"
+  appcast 'https://github.com/ApolloZhu/Dynamic-Dark-Mode/releases.atom'
+  name 'dynamic-dark-mode'
+  homepage 'https://github.com/ApolloZhu/Dynamic-Dark-Mode'
+
+  app 'dynamic.app'
```
Comment (ApolloZhu) [inline Casks/dynamic-dark-mode.rb]:
```suggestion
  app 'Dynamic.app'
```

File: Casks/dynamic-dark-mode.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'dynamic-dark-mode' do
+  version '1.0.6'
+  sha256 '2e82507ad65cf4eda208a8f41b4fe54944d0c04d15d1a49d1bcf546c65b7454d'
+
+  url "https://github.com/ApolloZhu/Dynamic-Dark-Mode/releases/download/#{version}/Dynamic-#{version}.zip"
+  appcast 'https://github.com/ApolloZhu/Dynamic-Dark-Mode/releases.atom'
+  name 'dynamic-dark-mode'
```
Comment (ApolloZhu) [inline Casks/dynamic-dark-mode.rb]:
```suggestion
  name 'Dynamic Dark Mode'
```

File: Casks/dynamic-dark-mode.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  homepage 'https://github.com/ApolloZhu/Dynamic-Dark-Mode'
+
```
Comment (ApolloZhu) [inline Casks/dynamic-dark-mode.rb]:
```suggestion

  depends_on macos: '>= :mojave'

```

Comment (dgisser) [reply Casks/dynamic-dark-mode.rb]: nice catch

**Comment by ApolloZhu** [comment]
@vitorgalvao is there a way we can update Travis to use Mojave?

**Comment by dgisser** [comment]
@vitorgalvao I see that os x 10.14 isn't supported by Travis yet (https://docs.travis-ci.com/user/reference/osx/#os-x-version). Do you think for now we could set it so that if it's travis it passes, otherwise check that for users they are running mojave?

File: Casks/dynamic.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://github.com/ApolloZhu/Dynamic-Dark-Mode/releases/download/#{version}/Dynamic-#{version}.zip"
+  appcast 'https://github.com/ApolloZhu/Dynamic-Dark-Mode/releases.atom'
+  name 'Dynamic Dark Mode'
+  homepage 'https://github.com/ApolloZhu/Dynamic-Dark-Mode'
+
+  depends_on macos: '>= :mojave'
+
+  app 'Dynamic.app'
+
+  uninstall pkgutil: 'io.github.apollozhu.Dynamic'
```
Comment (vitorgalvao) [inline Casks/dynamic.rb]: This doesn’t look right. It’s an `app`, not a `pkg`.

**Comment by vitorgalvao** [comment]
Don’t add Travis specific conditionals. Please be patient. It’ll be merged when it is.

**Pull request merged**